language/move-command-line-common/src/env.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+pub const KEEP_TMP: &str = "KEEP";
```

Comment:
Why this one is here, would it be better to move it to move-lang/tests/move_check_testsuite.rs?